content/introduction/getting-started.md

```diff
@@ -226,8 +226,8 @@ end
 ```
 
 This is an empty action that doesn't do anything special.
-Each action in Hanami is defined by [its own class](https://en.wikipedia.org/wiki/Single_responsibility_principle), which makes it simple to test and work on.
-And, each action has a corresponding view, which is also defined by its own class.
+Each action in Hanami is defined by [its class](https://en.wikipedia.org/wiki/Single_responsibility_principle), which makes it simple to test.
```

Comment:
```suggestion
Each action in Hanami is defined by [a single class](https://en.wikipedia.org/wiki/Single_responsibility_principle), which makes it simple to test.
```
I think this suggestion gets to the point we're trying to make here. (That it's different than Rails where you can have 7+ actions in a class)